### ISSUE TITLE
Decrypt some client.dll at old engines to get more features work

### ISSUE THREAD

**Issue by SmileyAG** [YaLTeR/BunnymodXT#301]
That would fixes a known issue when BXT huds does not work in HL: Blue Shift at build 4554 (GoldSrc Package 2.3)

Open-source algorithm for decrypt **client.dll**, **hw.dll**, **sw.dll** from GoldSrc: https://www.unknowncheats.me/forum/counterstrike-1-5-1-6-and-mods/443755-hldlldec-decrypting-libraries-versions.html (compiled + orig. sources: http://aluigi.altervista.org/papers/hldlldec.zip)

Some information from hoboken:

![hobo](https://user-images.githubusercontent.com/58108407/170831958-c0765907-2076-4e21-ad65-be1fc0f1d0be.PNG)

**Comment by YaLTeR** [comment]
> Open-source algorithm for decrypt client.dll, hw.dll, sw.dll from GoldSrc

BXT doesn't need this; the way it works for hw.dll is it waits for hl.exe to decrypt it and then just hooks hl.exe as if it was hw.dll, since it now contains all of the patterns.

**Comment by hobokenn** [comment]
same question here: what's the point of fixing this? i specifically made gsp 2.4 to avoid this issue

**Comment by SmileyAG** [comment]
> same question here: what's the point of fixing this? i specifically made gsp 2.4 to avoid this issue

Yeah, I don't see the need for this at the moment either, speedrunners should use GSP 2.4 instead, so I'll close it for now.
